extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -253,54 +293,49 @@ public function doWikiForm() {
 					$info = $this->doUpdateTags( $wgRequest );
 					break;
 				case "findtags":
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce this "switch/case" number of lines from 14 to at most 10, for example by extracting code into function. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)